securecn/resource_portshift_trusted_signer.go

```diff
@@ -186,8 +186,8 @@ func getTrustedSignerKeysFromConfig(d *schema.ResourceData) []*model2.TrustedSig
 	for k, v := range keysMap {
 		value := v.(string)
 		key := &model2.TrustedSignerKey{
-			Key:  &k,
-			Name: &value,
+			Key:  &value,
```

Comment:
'name' is the key and 'key' is the value?

Reply:
ok i looked and it looked like you are right..... weird

Reply:
yes.
for example: name = key1,
                      key = "-----BEGIN PRIVATE KEY----- asdlkl;a123;olawsd3124" ....